lib/rules/indent.js

```diff
@@ -170,7 +170,7 @@ module.exports = {
          * 1. Create a hashmap (`desiredOffsets`). The keys are all the tokens and comments in the file, and the values are objects
          *    containing information for a specific offset, measured in indent levels, from a either a specific token or the first column.
          *    For example, an element in an array will have {offset: 1, from: openingCurly} to indicate that it is offset by one indentation
```

Comment:
I'm really sorry to be a pain, but "openingCurly" on this line is also problematic. I should have been more explicit in the last review, sorry.